doc/project/checklist.md

```diff
@@ -117,28 +122,29 @@ Any lint waiver files have been reviewed.
 A CDC checking run has been set up (if tooling is available).
 The CDC checking run shows no must-fix errors, and a waiver file has been created.
 
-### FPGA_TIMING
+If there is no CDC flow at the block-level, this requirement can be waived.
 
-The IP block is synthesized as part of the continuous integration checking and meets timing there.
+### RDC_SETUP
 
-### CDC_SYNCMACRO
+An RDC checking run has been set up (if tooling is available).
+The RDC checking run shows no must-fix errors, and a waiver file has been created.
 
-All CDC synchronization flops use behavioral synchronization macros (e.g. `prim_flop_2sync`) not manually created flops.
+If there is no RDC flow at the block-level, this requirement can be waived.
 
-### SEC_CM_DOCUMENTED
+### AREA_CHECK
 
-Any appropriate security counter-measures have been documented and reviewed.
-The implementation of security countermeasures can be delayed until D3 if:
-- The addition of those countermeasures would not negatively impact PPA by more than 10%, and
-- The software interface would not be materially affected by those countermeasures
+An area check has been completed either with an FPGA or ASIC synthesis flow.
 
-Where the area impact of countermeasures can be reliably estimated, it is acceptable to insert dummy logic at D2 in order to meet the above criteria.
+### TIMING_CHECK
 
-### SEC_RND_CNST
+A timing check has been completed either with an FPGA or ASIC synthesis flow.
```

Comment:
We only run and publish these numbers from the synthesis flow, right?

Reply:
Yeah the dashboard only publishes the numbers from the ASIC flow at the moment.
The checklist is supposed to be generic however, hence we've mentioned FPGA or ASIC synthesis flow without referring to a specific tool.